Makefile

```diff
@@ -179,6 +209,14 @@ else
 	docker push $(IMAGE_TAG_LATEST)
 endif
 
+.PHONY: push-latest-scheduler
+push-latest-scheduler:
+ifdef CI
+	docker manifest push --purge $(SCHEDULER_IMAGE_TAG)
```

Comment:
For local development doing a "docker build ..." or azschedulerextender-container is enough.
In this PR I did not attempt to figure out final deployment of extender (i.e. how we will deploy when we preview\GA) and I am not sure if we will push the driver and the extender together or separately. That is why I have not touched the push-manifest target yet. If you want I can get rid of the ifdef CI portion to avoid confusion?

Reply:
Yeah, I believe this would be a good idea. It would also be good to leave a TODO comment regarding finalizing release & distribution plans.

Reply:
added. thanks!